quickstarts/microsoft.apimanagement/api-management-create-with-external-redis-cache/main.bicep

```diff
@@ -0,0 +1,98 @@
+
+@description('The location to deploy our resources to. Default is location of resource group')
+param location string = resourceGroup().location
+
+@description('The name of the APIM instance')
+param apimName string = 'apim-${uniqueString(resourceGroup().id)}'
+
+@description('The instance size of this API Management service. Must be 0 for Consumption')
+@allowed([
+  0
+])
+param skuCount int = 0
+
+@description('The pricing tier of this API Management service.')
+@allowed([
+  'Consumption'
+  'Developer'
+  'Standard'
+  'Premium'
+])
+param sku string = 'Consumption'
+
+@description('The name of the publisher')
+@minLength(1)
+param publisherName string
+
+@description('The email of the publisher')
+@minLength(1)
+param publisherEmail string
+
+@description('The name of the Azure Cache for Redis instance to deploy')
+param redisCacheName string = 'redis-${uniqueString(resourceGroup().id)}'
+@description('Specify the pricing tier of the new Azure Redis Cache.')
+@allowed([
+  'Basic'
+  'Standard'
+  'Premium'
+])
+param redisCacheSKU string = 'Basic'
+
+@description('Specify the family for the sku. C = Basic/Standard, P = Premium.')
+@allowed([
+  'C'
+  'P'
+])
+param redisCacheFamily string = 'C'
+
+@description('Specify the size of the new Azure Redis Cache instance. Valid values: for C (Basic/Standard) family (0, 1, 2, 3, 4, 5, 6), for P (Premium) family (1, 2, 3, 4)')
+@allowed([
```

Comment:
use minValue and maxValue instead of allowed

Reply:
Changed to minValue and maxValue